rbc/omniscidb.py

```diff
@@ -370,6 +370,13 @@ def thrift_call(self, name, *args, **kwargs):
             m = re.match(r'Could not bind *', msg.error_msg)
             if m:
                 raise OmnisciServerError(msg.error_msg)
+            m = re.match(r'Runtime extension functions registration is disabled.',
+                         msg.error_msg)
+            if m:
+                msg = (f"{msg.error_msg} Please use --enable-runtime-udf"
+                       " and/or --enable-table-functions")
```

Comment:
```suggestion
                msg = (f"{msg.error_msg} Please use server options --enable-runtime-udf"
                       " and/or --enable-table-functions")
```
or similar so that users will have some glues where these options should be used.